dist/21-526EZ-schema.json

```diff
@@ -2225,24 +2225,20 @@
         "homelessness": {
           "type": "object",
           "required": [
-            "hasPointOfContact"
+            "isHomeless"
           ],
           "properties": {
-            "hasPointOfContact": {
-              "type": "boolean",
-              "default": false
+            "isHomeless": {
+              "type": "boolean"
             },
             "pointOfContact": {
               "type": "object",
-              "required": [
-                "pointOfContactName",
-                "primaryPhone"
-              ],
               "properties": {
                 "pointOfContactName": {
                   "type": "string",
+                  "minLength": 1,
                   "maxLength": 100,
-                  "pattern": "^([a-zA-Z0-9\\-'.#][a-zA-Z0-9\\-'.# ]?)*$"
+                  "pattern": "^([a-zA-Z0-9-/]+( ?))*$"
```

Comment:
Why update this? Is this the updated pattern in the swagger docs?

Reply:
Yeah, exactly. I guess they changed it on the sly. 🤷‍♂️

Reply:
Bugger. Looks like we're gonna have to do _another_ thorough sweep of the whole bloody schema to make sure it matches the swagger docs before we're done.